qldbdriver/transaction.go

```diff
@@ -70,7 +70,27 @@ func (txn *transaction) execute(ctx context.Context, statement string, parameter
 	if err != nil {
 		return nil, err
 	}
-	return &Result{ctx, txn.communicator, txn.id, executeResult.FirstPage.Values, executeResult.FirstPage.NextPageToken, 0, txn.logger, nil, nil}, nil
+
+	// create IOUsage and copy the values returned in executeResult.ConsumedIOs
+	var ioUsage *IOUsage = nil
+	if executeResult.ConsumedIOs != nil {
+		readIOs := *executeResult.ConsumedIOs.ReadIOs
```

Comment:
Does this actually make a copy of the int64? It should. Feels like a lot of code to achieve copying three int64s.

Reply:
Simplified to:

```
*ioUsage.readIOs += *executeResult.ConsumedIOs.ReadIOs
```